components/automate-cluster-ctl/habitat/plan.sh

```diff
@@ -18,11 +18,11 @@ pkg_deps=(
   core/gzip
   core/jq-static
   core/openssh
-  core/openssl
+  core/openssl/1.0.2zb/20220311111046
   core/make
   core/curl
-  core/rsync
-  core/terraform/0.14.8/20210826165930
+  core/rsync/3.2.3/20220817235706
+  schaudha/terraform/1.5.7/20230921122623
```

Comment:
can we make the final changes for the terrafrom package ?

Reply:
I have pushed the changes which have the new terraform core/terraform1 package.